agent/bls.mjs

```diff
@@ -1334,9 +1400,9 @@ async function do_sign_messages_s2s(
     );
 }
 
-async function do_sign_u256( u256, details, fn ) {
+export async function do_sign_u256( u256, details, fn ) {
```

Comment:
Please, modify function so its maximum size will be 100 codelines:
https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/code-guidelines.md#function-length

Reply:
Done